### DISCUSSION TITLE
ETA for relations

### DISCUSSION THREAD

**Discussion by nicooprat** [CodeDredd/pinia-orm#670] [General]
Hi there,

Thanks for your work.

We're about to finish our migration from Vuex to Pinia. Looks like relations are still not supported though: https://github.com/CodeDredd/pinia-orm#migration-from-vuex-orm

Any plan to support it? Are you following vuex-orm-next development timeline (so I should ask there)?

Thanks!

**Comment by CodeDredd** [comment]
Hey and thanks ❤️ ,

Maybe the overview confused you a bit. Relations are supported. Only some specific ones not, like `morphToMany`, `morphedByMany`, `hasManyThrough`. Do you need one of those ?

**Reply by nicooprat** [reply]
Yes I should have been more precise, I actually need `hasManyThrough` ;)

**Reply by CodeDredd** [reply]
@nicooprat For a cup of ☕ i can make this feature even faster 😉 It's hard without it believe me 😆

**Reply by nicooprat** [reply]
Sure! Actually we decided to spend our annual team budget rest to support open-source, so I just added pinia-orm to the list :)

**Reply by CodeDredd** [reply]
How fast do you need the relation ?

**Comment by nicooprat** [comment]
No rush, it’s open source after all :)

In the coming weeks or months it would be cool!

Anyway we have to first check that everything else related to the ORM works too.

--
Nico Prat
***@***.***
@nicooprat

**Reply by CodeDredd** [reply]
Well i have finished it already...so just keep informed for anything else 😉

**Reply by nicooprat** [reply]
Amazing 💪

**Comment by getriebesand** [comment]
Hi, are there any plans for morphToMany? Thank you

**Reply by CodeDredd** [reply]
Hi. Well i hadn't a plan yet for it. Was putting my spare time in documentation and v2 thoughts.
But now that some needs it, i can try implement it.
A cup of ☕ helps me a lot to get it done faster 😄

**Reply by getriebesand** [reply]
yes please :) i have used this relation in vuex-orm but there it is a little bit broken

**Reply by CodeDredd** [reply]
@getriebesand Thanks for the support! ❤️  Here is your reward 😉 https://github.com/CodeDredd/pinia-orm/pull/1060
